enabling-ossm.md

```diff
@@ -0,0 +1,183 @@
+# Steps to install ODH with OSSM
+
+## Prerequisites
+
+* Openshift cluster with enough karma (make sure to log in :))
+* CLI tools
+  * `kustomize` 
+  * `kubectl`
+  * (until operator changes are merged) `operator-sdk` v1.24.1
+
+* Installed operators
+  * Kiali
+  * Jaeger
+  * OSSM
+  * OpenDataHub
+  * Authorino
+  
+### Check if required operators are installed
+
+```sh
+kubectl get operators | awk -v RS= '/kiali/ && /jaeger/ && /servicemesh/ && /opendatahub/ && /authorino/ {exit 0} {exit 1}' || echo "Please install all required operators."
+```
+
+Install required operators
+
+  ```sh
+  createSubscription() {
+    local name=$1
+    local source=${2:-"redhat-operators"}
+    local channel=${3:-"stable"}
+
+    echo  "Create Subscription resource for $name"
+    eval "kubectl apply -f - <<EOF
+  apiVersion: operators.coreos.com/v1alpha1
+  kind: Subscription
+  metadata:
+    name: $name
+    namespace: openshift-operators
+  spec:
+    channel: $channel
+    installPlanApproval: Automatic
+    name: $name
+    source: $source
+    sourceNamespace: openshift-marketplace
+  EOF"    
+  }
+  ```
+  
+
+  ```sh
+  createSubscription "kiali-ossm"
+  createSubscription "jaeger-product"
+  createSubscription "servicemeshoperator"
+  # createSubscription "opendatahub-operator" "community-operators"
+  # temp, until operator changes are merged.
+  operator-sdk run bundle quay.io/cgarriso/opendatahub-operator-bundle:dev-0.0.2 --namespace openshift-operators --timeout 5m0s
+  createSubscription "authorino-operator" "community-operators" "alpha"
+  ```
+
+Note that you may need to go into the `Installed Operators` tab in the console to manually finalize the install of the authorino operator.
+
+## Setting up Open Data Hub Project
+
+### Create Kubeflow Definition
+
+```sh
+cat <<'EOF' > odh-minimal.ign.yaml
```

Comment:
File name does not match with the one used in `kubectl apply` a little lines later.

```suggestion
cat <<'EOF' > odh-minimal.yaml
```

Reply:
Or the other way around - fix kubectl apply. The reason for having this as `*ign` is to allow some global gitignore rules so you won't accidentally commit such files. I just pushed the fix for it. Thanks for spotting and suggesting. Fixed in a78e490